docs/operator-guide/cd/manual-approval.md

```diff
@@ -0,0 +1,109 @@
+# Manual Approval in Deploy Pipelines
+
+Since the 3.10.0 release, KubeRocketCI features manual approval task in deploy pipelines, which stops the pipeline execution until the user makes a decision or pipeline is timed out. This feature allows to integrate a quality control checkpoint in your pipelines, reducing risks when deploying applications to production environments.
+
+## Manual Approval Structure
+
+A deploy pipeline with a manual approval step is a typical Tekton pipeline but with an [ApprovalTask](https://github.com/epam/edp-tekton/blob/48f0d18ecfec049618999dabd9c8cbc892ae3508/charts/pipelines-library/templates/pipelines/cd/deploy-with-approve.yaml#L61) custom resource included. The custom deploy pipeline usage flow involves the following steps:
```

Comment:
why we use specific commit instead of branch or tag?

Reply:
do we need description about custom pipeline here?

Reply:
Creating pipeline with approval task is basically creating a custom pipeline, so I guess it would be prudent to explain what it is.